### PR TITLE
Add mobile notification section for swaps

### PR DESCRIPTION
**Pull request by ok300**
This PR adds a short section in the swaps page, informing about the automatic swap notifications and their effect on swaps.

### CODE REVIEW

**Review by dangeross — APPROVED** [review]
LGTM

File: src/guide/receive_onchain.md [hunk 1]
```diff
@@ -156,6 +156,16 @@ The process of receiving funds via an on-chain address is trustless and uses a s
...
+
+## Notifications
+
+Enabling [mobile notifications](payment_notification.md) in your app will automatically register it for swap notifications.
+
+This means that, when the user performs a swap-in (receive onchain), the app will
+
+- automatically complete the swap in the background when the onchain transaction is confirmed, even if the app is closed
+- display an OS notification, informing the user of the received funds
+
```
Comment (erdemyerebasmaz) [inline src/guide/receive_onchain.md:168]: I find using brief sentences in documentation more effective. 

```suggestion
When the user initiates a swap-in (receive on-chain), the app will

- complete the swap when the on-chain transaction is confirmed,
- notify users of received funds via a push notification

even if the app is backgrounded or closed.
```

File: src/guide/receive_onchain.md [hunk 2]
```diff
@@ -156,6 +156,16 @@ The process of receiving funds via an on-chain address is trustless and uses a s
 1. Either by a preimage that is exposed when the Lightning payment is completed - this is the positive case where the swap was successful.
 2. Or by your node when the swap didn't complete within a certain timeout (216 blocks) - this is the negative case where your node will execute a refund (funds become refundable after 288 blocks). Refund will also be available in case the amount sent wasn't within the limits.
 
+
+## Notifications
+
+Enabling [mobile notifications](payment_notification.md) in your app will automatically register it for swap notifications.
```
Comment (erdemyerebasmaz) [inline src/guide/receive_onchain.md:162]: Either of these two
```suggestion
Enabling [mobile notifications](payment_notification.md) will register your app for swap notifications.
```
or
```suggestion
By enabling [mobile notifications](payment_notification.md), your app will be registered for swap notifications.
```

File: src/guide/receive_onchain.md (hunk 1 above)
Comment (ok300) [reply src/guide/receive_onchain.md:168]: I find the initial points more accurate and helpful for the user. The first highlights that it happens automatically, even if the app is closed. The second highlights its an OS notification (user-facing), whereas push notification is a technical implementation detail.

So for this snippet, I'd keep the original.

**Review by erdemyerebasmaz — APPROVED** [review]

**Pull request merged**